src/utils/compute.ts

```diff
@@ -39,3 +43,44 @@ export async function transformComputeFormToServiceComputePrivacy(
 
   return privacy
 }
+
+export async function transformDDOToAssetSelection(
```

Comment:
I like making this generic to all assets but when looking at this method, it probably should continue living in `utils/aquarius.ts`. Unless you have something in mind changing it to doing something specific to compute

Reply:
sure I'll move it, I did that because it has nothing to do with aquarius so i thought that it was more related to the compute, but you are right we can use for more generic purpose